jzp-code/kukdo/project/cloud-storage/cloud-storage-parent/cloud-storage-web/src/main/java/com/xiaomi/service/impl/AliyunStorageServiceImpl.java

```diff
@@ -0,0 +1,90 @@
+package com.xiaomi.service.impl;
+
+import com.aliyun.oss.ClientConfiguration;
+import com.aliyun.oss.OSS;
+import com.aliyun.oss.OSSClientBuilder;
+import com.xiaomi.service.StorageService;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.io.InputStreamResource;
+import org.springframework.core.io.Resource;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.InputStream;
+
+@Service("AliyunStorage")
+public class AliyunStorageServiceImpl implements StorageService {
+
+    @Value("${aliyun.oss.endpoint}")
+    private String endpoint;
+
+    @Value("${aliyun.oss.accessKeyId}")
+    private String accessKeyId;
+
+    @Value("${aliyun.oss.accessKeySecret}")
+    private String accesKeySecret;
+
+    @Value("${aliyun.oss.bucketName}")
+    private String bucketName;

```

Comment:
consider `@ConfigurationProperties`

Reply:
> consider `@ConfigurationProperties`

Thanks for the suggestion, I created a new class `AliyunProperties` for storing Aliyun configurations, thus separating the configuration properties from the business logic, using @ConfigurationProperties(prefix = "aliyun.oss") and @EnableConfigurationProperties( AliyunProperties.class).